contrib/cirrus/cirrus_yaml_test.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+"""
+Verify contents of .cirrus.yml meet specific expectations
+"""
+
+import sys
+import os
+import unittest
+import yaml
+
+# Assumes directory structure of this file relative to repo.
+SCRIPT_DIRPATH = os.path.dirname(os.path.realpath(__file__))
+REPO_ROOT = os.path.realpath(os.path.join(SCRIPT_DIRPATH, '../', '../'))
+
+
+class TestCaseBase(unittest.TestCase):
+
+    CIRRUS_YAML = None
+
+    def setUp(self):
+        with open(os.path.join(REPO_ROOT, '.cirrus.yml')) as cirrus_yaml:
+            self.CIRRUS_YAML = yaml.safe_load(cirrus_yaml.read())
+
+
+class TestDependsOn(TestCaseBase):
+
+    ALL_TASK_NAMES = None
+    SUCCESS_RELEASE = set(['success', 'release'])
+
+    def setUp(self):
+        super().setUp()
+        self.ALL_TASK_NAMES = set([key.replace('_task', '')
+                                   for key, _ in self.CIRRUS_YAML.items()
+                                   if key.endswith('_task')])
+
+    def test_dicts(self):
```

Comment:
Just a heads-up for next time, no need to change it here: python `unittests` executes in alphabetical order, not module-defined order. If (improbably) success/release_task are missing from `.cirrus.tml`, the error messages will be most unhelpful because `test_depends` runs before this one.

Reply:
Thanks for taking a look.

Ya, they're def. not noob-friendly tests at this point.  Tossed together on moment's notice for specific purpose.  There are other checks that could be added, so hopefully this (and other issues) can be addressed in a future PR.

(I'd add to your list: pick better variables than "name" and "task_name", they made me go slightly cross-eyed a few times)